api/app/utils/polygonize.py

```diff
@@ -30,12 +30,13 @@ def polygonize_in_memory(geotiff_filename, layer, field) -> ogr.Layer:
     source: gdal.Dataset = gdal.Open(geotiff_filename, gdal.GA_ReadOnly)
 
     source_band = source.GetRasterBand(1)
+    nodata_value = source_band.GetNoDataValue()
```

Comment:
Nice, didn't know that existed, I wonder how gdal decides on what a no data value is.

Reply:
It's actually written into the header of every raster file (if it has a NoData value). You can see it with gdalinfo